src/TropicalGeometry/linear_space.jl

```diff
@@ -126,7 +126,7 @@ end
 @doc raw"""
     tropical_linear_space(k::Int, n::Int, p::Vector{<:TropicalSemiringElem}; weighted_polyhedral_complex_only::Bool=false)
 
-Return a tropical linear space from a tropical Pluecker vector with indices `AbstractAlgebra.combinations(1:n,k)` and values `p`.  If `weighted_polyhedral_complex==true`, will not cache any extra information.
+Return a tropical linear space from a tropical Pluecker vector with indices `combinations(n,k)` and values `p`.  If `weighted_polyhedral_complex_only==true`, will not cache any extra information.
 
 # Examples
```

Comment:
@YueRen This function is rather problematic. Is this really needed, and where? Could we deprecate it?

Reply:
Will the new implementation of `combinations` return the subsets in the same order as `minors`?  That is the sole reason why I am using `AbstractAlgebra.combinations` (and `AbstractAlgebra.minors`).